packages/utils/src/dom/get-element-by-url.ts

```diff
@@ -29,45 +35,55 @@ const getSrcsetUrls = (srcset: string): string[] => {
  * @param {string} url - URL that the element has to contain.
  */
 export const getElementByUrl = (dom: HTMLDocument, url: string): HTMLElement | null => {
-    // TODO: Cache dom.querySelectorAll?.
-    const elements = dom.querySelectorAll('[href],[src],[poster],[srcset]').filter((element: any) => {
+    const elements = dom.querySelectorAll('a[href],link[href],audio[src],iframe[src],script[src],source[src],track[src],frame[src],img[src],video[poster],img[srcset],source[srcset]');
```

Comment:
Minor request: would you mind sorting these alphabetically? It will make it easier to see anything we might have missed in the future. Especially having items like `source[src]` and `source[srcset]` next to each other. Thanks!